pkg/windows/windows.go

```diff
@@ -932,7 +932,10 @@ func (vm *windows) isContainersFeatureEnabled() (bool, error) {
 
 // isWindowsDefenderEnabled returns true if the Windows Defender antivirus/firewall is installed on the Windows instance
 func (vm *windows) isWindowsDefenderEnabled() (bool, error) {
-	command := "(Get-Service | where {$_.DisplayName -Like 'Windows Defender*'}).Count -gt 0"
+	// Check if process associated with Windows Defender does not exist. Reference:
+	// https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-compatibility?view=o365-worldwide#use-windows-powershell-to-confirm-that-microsoft-defender-antivirus-is-running
+	winDefProcess := "MsMpEng.exe"
+	command := "(Get-Process -Name " + winDefProcess + " -ErrorAction SilentlyContinue) -eq $null"
 	out, err := vm.Run(command, true)
```

Comment:
If there is no process with a name equal to MsMpEng.exe that means the Microsoft defender antivirus is not running (and likewise, if there is a MsMpEng.exe process then Defender is running). I don't follow the ask here.

Reply:
Correct. Manually debugging this code to expose my concern.

`out` is equal to `"True"` if there is no process with a name equal to `MsMpEng.exe`, which means Microsoft defender antivirus **is not** running and the function returns `strings.TrimSpace(out) == "True"` that evaluate to `true`. 

So, `isWindowsDefenderRunning()` returning `true`  means the Microsoft defender antivirus **is** running


IMO, `isWindowsDefenderRunning()` should returns `false` if there is no process with a name equal to `MsMpEng.exe`.


```go
        ...
	out, err := vm.Run(command, true)
	if err != nil {
		return false, errors.Wrap(err, "error checking if Windows Defender is enabled")
	}
	return strings.TrimSpace(out) == "True", nil
```

Reply:
Thanks for the explanation, I see now. I need to flip the condition i.e. `return strings.TrimSpace(out) == "False"`

Reply:
You may not need a conditional statement at all in the PowerShell command. You can compare with the output of `Get-Process -Name MsMpEng.exe` directly.

Reply:
The condition seems to be the easiest way. Comparing with the output would require Golang-based Powershell error discernment, which I'd like to avoid.